stacks-signer/src/signer.rs

```diff
@@ -1417,6 +1477,10 @@ impl Signer {
                     "{self}: updated DKG value from {old_dkg:?} to {:?}.",
                     self.approved_aggregate_public_key
                 );
+                if let Err(_) = self.load_saved_state_for_aggregate_key(approved_aggregate_key) {
+                    warn!("{self}: Failed to load saved state for key {approved_aggregate_key}");
+                }
+                self.send_dkg_results(&approved_aggregate_key);
```

Comment:
Should you be doing this if you failed to load your saved state?

Reply:
Perhaps not, especially since it's not critical that all signers send these DKG results. So I can definitely condition the sending on these on the success of the load.

Reply:
Updated so it's conditioned on a successful state load.